queries/snakemake/indents.scm

```diff
@@ -9,8 +9,18 @@
 ((rule_inheritance) @indent.begin
   (#set! indent.immediate 1))
 
+((rule_import
+  "with"
+  ":") @indent.begin
+  (#set! indent.immediate 1))
+
 ((module_definition) @indent.begin
   (#set! indent.immediate 1))
 
 ((directive) @indent.begin
   (#set! indent.immediate 1))
+
+; end indentation after last parameter node (no following ',')
+(directive_parameters
+  (_)*
+  (_) @indent.end)
```

Comment:
It seems like this is only for the last child of `directive_parameters`. In that case, something like this will be better
```suggestion
(directive_parameters
  (_) @indent.end .)
```